nilearn/datasets/func.py

```diff
@@ -2168,24 +2168,44 @@ def _reduce_confounds(regressors, keep_confounds):
 
 
 @fill_doc
-def fetch_language_localizer_demo_dataset(data_dir=None, verbose=1):
+def fetch_language_localizer_demo_dataset(
+    data_dir=None, verbose=1, legacy_output=True
+):
     """Download language localizer demo dataset.
 
     Parameters
     ----------
     %(data_dir)s
     %(verbose)s
+    legacy_output: bool, default=True
+
+        .. versionadded:: 0.11.0
+        .. deprecated:: 0.11.0
+
+            Starting from version 0.13.0
+            the ``legacy_ouput`` argument will be removed
+            and the fetcher will always return
+            a ``sklearn.datasets.base.Bunch``.
+
 
     Returns
     -------
+    data : sklearn.datasets.base.Bunch
```

Comment:
I think that's suppose to be

```suggestion
    data : :class:`sklearn.utils.Bunch`
```
and can link to it with a sphinx role

Reply:
Thanks I was looking for the right way to write this but was failing.